googlemock/include/gmock/internal/gmock-port.h

```diff
@@ -60,6 +60,27 @@
 # error "At least Visual C++ 2015 (14.0) is required to compile Google Mock."
 #endif
 
+// GMOCK_API_ qualifies all symbols that must be exported. The definitions below
+// are guarded by #ifndef to give embedders a chance to define GMOCK_API_ in
+// gmock/internal/custom/gmock-port.h
+#ifndef GMOCK_API_
+
+#ifdef _MSC_VER
+# if GMOCK_LINKED_AS_SHARED_LIBRARY
+#  define GMOCK_API_ __declspec(dllimport)
+# elif GMOCK_CREATE_SHARED_LIBRARY
+#  define GMOCK_API_ __declspec(dllexport)
+# endif
+#elif __GNUC__ >= 4 || defined(__clang__)
+# define GMOCK_API_ __attribute__((visibility ("default")))
```

Comment:
I believe the reason why `GTEST_API_` macro was there because of the fact that mocking is a subset of `googletest` and should be served under as its public API. The whole point of `__attribute__((visibility ("default"))` is to expose those symbols for dynamic linkage, done either way makes no difference but separates one from another. In my opinion `GTEST_API_` should not be removed, again not an expert, just an opinion.